tests/ClientBuilderTest.php

```diff
@@ -188,13 +188,10 @@ public function optionsDataProvider(): array
             ['setSerializeAllObjects', false],
             ['setSampleRate', 0.5],
             ['setMbDetectOrder', ['foo', 'bar']],
-            ['setAutoLogStacks', false],
+            ['setAttachStacktrace', true],
             ['setContextLines', 0],
             ['setEnableCompression', false],
-            ['setCurrentEnvironment', 'test'],
-            ['setEnvironments', ['default']],
-            ['setExcludedLoggers', ['foo', 'bar']],
-            ['setExcludedExceptions', ['foo', 'bar']],
```

Comment:
You didn't remove `setExcludedExceptions`, that's an error?

Reply:
Fixed